app/src/main/java/com/example/campusguide/calendar/Login.kt

```diff
@@ -89,31 +86,31 @@ class Login constructor(
                 updateUI()
             }
         } catch (e: ApiException) {
-            Log.w(TAG, "signInResult:failed code=" + e.statusCode)
+            Log.w(Constants.TAG, "signInResult:failed code=" + e.statusCode)
         }
     }
 
     fun signIn() {
         val signInIntent: Intent = mGoogleSignInClient.signInIntent
-        activity.startActivityForResult(signInIntent, RC_SIGN_IN)
+        activity.startActivityForResult(signInIntent, Constants.RC_SIGN_IN)
         // Ask for calendar permissions
         getCalendarPermissions()
     }
 
     fun signOut() {
         mGoogleSignInClient.signOut()
         userEmail = null
-        Toast.makeText(activity, "Logged Out", Toast.LENGTH_LONG).show()
+        Toast.makeText(activity, Constants.LOGGED_OUT_TOAST, Toast.LENGTH_LONG).show()
         // Change menu item title for Login asking to login to an account
-        getNavView().menu.findItem(R.id.login_button).title = "Login to an Account"
+        getNavView().menu.findItem(R.id.login_button).title = Constants.LOGIN_TO_ACCOUNT
         // Disable view for Calendar menu item
         getNavView().menu.findItem(R.id.calendar).isVisible = false
     }
 
     private fun updateUI() {
-        Toast.makeText(activity, "Logged into $userEmail", Toast.LENGTH_LONG).show()
+        Toast.makeText(activity, Constants.LOGGED_INTO_TOAST + "$userEmail", Toast.LENGTH_LONG).show()
```

Comment:
Do `"${Constants.LOGGED_IN_TOAST} $userEmail"` instead of adding two strings together. Applies to any other places where you add strings

Reply:
Will do!